src/scene/materials/standard-material.js

```diff
@@ -183,6 +183,31 @@ let _params = new Set();
  * @property {number} clearCoatBumpiness The bumpiness of the clear coat layer. This value scales
  * the assigned main clear coat normal map. It should be normally between 0 (no bump mapping) and 1
  * (full bump mapping), but can be set to e.g. 2 to give even more pronounced bump effect.
+ * @property {boolean} useIridescence Enable thin-film iridescence.
+ * @property {Texture|null} iridescenceMap The per-pixel iridescence intensity. Only used when useIridescence is enabled.
```

Comment:
I need to add this to the coding standards but all JSDoc blocks should stick to 100 columns.